mobile/src/components/course/styles.ts

```diff
@@ -0,0 +1,46 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor, AppFontFamily } from '~/common/enums/enums';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 20,
+    backgroundColor: AppColor.BACKGROUND.GRAY_400,
+  },
+  h1: {
+    color: AppColor.TEXT.GRAY_100,
+    fontFamily: AppFontFamily.INTER_600,
+    fontSize: 24,
+    lineHeight: 32,
+    marginBottom: 15,
+  },
+  h2: {
+    color: AppColor.TEXT.GRAY_100,
+    fontFamily: AppFontFamily.INTER_600,
+    fontSize: 18,
+    lineHeight: 24,
+    marginBottom: 15,
+  },
+  image: {
+    width: '100%',
+    height: 200,
+    resizeMode: 'contain',
+    borderRadius: 12,
+    marginBottom: 30,
+  },
+  text: {
+    color: AppColor.TEXT.GRAY_100,
+    fontFamily: AppFontFamily.INTER_400,
+    fontSize: 14,
+    lineHeight: 19.5,
+  },
+});
+
+const tagsStyles = {
```

Comment:
Can't we fix margin before numbers?
<img width="65" alt="Screenshot 2022-08-26 at 08 02 08" src="https://user-images.githubusercontent.com/64521658/186833076-2dbaba99-7660-4f7a-96a9-5e60ea103bb2.png">

Reply:
Yes, I will fix this in next commit.